index.html

```diff
@@ -0,0 +1,157 @@
+    <title>Sohaib Capstone</title>
+</head>
+<body>
+    <header>
+        <nav class="black-navbar">
+            <a href="#">
+                <i class="fa-brands fa-facebook-f"></i>
+            </a>
+            <a href="#">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="#">Logout</a>
+            <a href="#">MyPage</a>
+            <a href="#" class="last">Dubai</a>
+        </nav>
+
+        <nav class="white-navbar">
+            <div class="desk-logo">
+                <a href="index.html">
+                    <img src="./images/index logo.png" alt="Index Page Logo">
+                </a>
+            </div>
+            <ul class="nav-itmes">
+              <li>
+                <a class="nav-links first" href="about.html">About</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#program">Program</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#">Join</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#sponser">Sponser</a>
+              </li>
+              <li>
+                <a class="nav-links" href="#">News</a>
+              </li>
+              <li>
+                <a href="#">
+                    <button class="nav-btn">Campaign</button>
+                </a>
+              </li>
+            </ul>
+        </nav>
+        <nav class="mobile-nav">
+            <div class="close-btn">
+                <i class="fa-solid fa-xmark"></i>
+            </div>
+          <ul class="nav-list">
+            <li>
+              <a class="nav-links" href="about.html">About</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#program">Program</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#">Join</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#">Sponser</a>
+            </li>
+            <li>
+              <a class="nav-links" href="#">News</a>
+            </li>
+            <li>
+              <a href="#">
+                  <button class="mobile-nav-btn">Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </nav>  
+        <div class="menu-btn">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+        </div >
+     <div class="header-info">
+        <div class="summit-header">
+            <img src="./images/summit.png" alt="First Header">
+        </div>
+        <p class="summit-paragraph">
+            A fun festival where people from more than 80 countries who believe in the value of sharing and openness and create positive change gather and share ideas will be held in Korea in October.
+        </p>
+        <h1 class="date">2015.2.10 (Mon) ~ 13 (Ths)</h1>
+        <p class="place location">
+          @ National Museum of Korea, Art Center Nabi, etc.</p>
+     </div>
```

Comment:
- [ ] This is the top section of your home page and would be better placed outside the `<header>` tag and in a proper `<section>` tag. Kindly apply these changes.

Reply:
Done